modules/i18n/src/translations/si-LK/portals/console.ts

```diff
@@ -8784,7 +8784,8 @@ export const console: ConsoleNS = {
                             },
                             advanceSearch: {
                                 placeholder: "පරිශීලක නාමයෙන් සොයන්න"
-                            }
+                            },
+                            disabledSecondaryStoreInfo: "බාහිර පරිශීලක වෙළඳසැල් වෙත තොග ආනයනය මේ මොහොතේ ලබා ගත නොහැක."
```

Comment:
Shall we rephrase this ?

Reply:
I rephrased this as follows. WDYT?
"බාහිර පරිශීලක වෙළඳසැල් වෙත පරිශීලක තොග ආනයනය මේ මොහොතේ ලබා දිය නොහැක."